IntegrationTests/tests_04_performance/test_01_resources/run-nio-alloc-counter-tests.sh

```diff
@@ -18,25 +18,24 @@ here="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 tmp_dir="/tmp"
 
-function die() {
-    echo >&2 "ERROR: $*"
-    exit 1
-}
-
 while getopts "t:" opt; do
     case "$opt" in
         t)
             tmp_dir="$OPTARG"
             ;;
-        \?)
```

Comment:
why this change?

Reply:
My thinking was most of the examples of `getopts` I've seen use `*` to handle the unknown case, as it's idiomatic with the default branch in a `case` block in general.

With `\?`, you have to remember that `getopts` will set the flag variable to `?` if it doesn't recognise a flag, so it's the unknown case you're handling with this branch. You also have to remember that you don't need a separate default branch for the `case`, because the variable will always be `?` if a flag is unknown.

Compared to a list of flags and then a default `*` case, that seemed less clear and harder to remember at first glance.

I don't feel strongly about it though, happy to revert it.